python/pyspark/mllib/tests.py

```diff
@@ -25,7 +25,8 @@
 import array as pyarray
 from time import time, sleep
 
-from numpy import array, array_equal, zeros, inf, all, random
+from numpy import (
+    array, array_equal, zeros, inf, random, exp, dot, all, mean)
```

Comment:
Just curious, we don't need `(...)`, right? Btw, it is safer to write `import numpy as np` and then use `np.array` in the code. For example, `all` is a built-in method of Python. We can update this in a separate PR.